src/VsixTesting.Installer/Program.cs

```diff
@@ -329,7 +329,7 @@ public static IVsExtensionManager Create(ExternalSettingsManager externalSetting
         }
 
         public void UpdateLastExtensionsChange()
-            => Obj.UpdateLastExtensionsChange();
+            => Obj.UpdateLastExtensionsChange(true);
```

Comment:
❔ Is this still going to work with older versions?

Reply:
I don't know. We don't have an old assembly version to test. Maybe they removed an overload, maybe they added the parameter.
Other than using package versions for that, could be to: try without parameter, catch (RuntimeBinderException) and to with parameter. Reflecting on that method if it has or has not any known parameters (0 or 1 bool) seems a bit overkill, if it would have no idea what to do in the else-branch anyway.